webapp/Connector/src/app/store/StudyProducts.js

```diff
@@ -94,6 +103,22 @@ Ext.define('Connector.app.store.StudyProducts', {
         this._onLoadComplete();
     },
 
+    getById : function(id) {
```

Comment:
I tried the route of adding a flag, but I had a difficult time finding where to tell it not to display based on that flag.
Also, this route guarantees that these non-products are not included in filter facet views, and by anything that counts products. The way I was thinking about these (NONE, COMING_SOON, NOT_USED) is that they are semantically not products, they just happen to have the same fields. So keeping them separate from the rest of the product data just seemed prudent.